test/TestUtilities/Test.Utility/Signing/SigningTestUtility.cs

```diff
@@ -580,7 +580,7 @@ public bool Equals(CertificateHashAllowListEntry x, CertificateHashAllowListEntr
                     return false;
                 }
 
-                return x.VerificationTarget == y.VerificationTarget &&
+                return x.Target == y.Target &&
                     x.FingerprintAlgorithm == y.FingerprintAlgorithm &&
                     string.Equals(x.Fingerprint, y.Fingerprint, StringComparison.Ordinal);
```

Comment:
Spoke to Damon about this. This comparison should also be `OrdinalIgnoreCase`. Please correct this.